.github/workflows/save_daily_metrics.yml

```diff
@@ -53,7 +51,7 @@ jobs:
       - shell: bash -l {0}
         name: Save Github Metrics
         env:
-          API_TOKEN_GITHUB: ${{ secrets.API_TOKEN_GITHUB }}
+          API_TOKEN_GITHUB: ${{ secrets.USAGE_STATS_ACCESS_TOKEN }}
```

Comment:
Did you have to generate a new token or something?

Reply:
Yes, this was a reference to a secret that existed on the `business` repo. We can't authenticate within the GHA as accessing metrics requires admin level read-only access, so this required generating a new token.